internal/driver/examples/kubernetes/csi-app.yaml

```diff
@@ -7,9 +7,13 @@ spec:
     - name: csi-example-container
       image: busybox
       volumeMounts:
-      - mountPath: "/data"
-        name: csi-example-volume
-      command: [ "sleep", "1000000" ]
+        - mountPath: "/data"
+          name: csi-example-volume
+      command: ["sleep", "1000000"]
+  tolerations:
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/control-plane
+      operator: Exists
```

Comment:
question: Do we want this toleration to be part of the example?

Reply:
yeah this will allow the pod to be scheduled on a CP node. As our Makefile is primarily designed for a single control plane node, chances are users will follow that and end up in a holding state waiting for the pod to be scheduled